examples/anthropic_stream.ts

```diff
@@ -0,0 +1,23 @@
+import { Anthropic } from "llamaindex";
```

Comment:
put them into a single directory

Reply:
Are you saying we should do a subdirectory level import? I think it would be a good idea to document, although not sure we want to make it standard across our examples just yet.